packages/eslint-config/index.js

```diff
@@ -222,6 +222,10 @@ const unicornRules = {
   'unicorn/prefer-node-protocol': 'off',
   // Conflict with `@silverhand/fp/no-mutating-methods` when calling `.slice().sort()`
   'unicorn/prefer-spread': 'off',
+  'unicorn/no-await-expression-member': 'off',
+  'unicorn/template-indent': 'off',
+  'unicorn/prefer-export-from': 'off',
```

Comment:
same here , should we follow the approach?

Reply:
logto/js/packages/client/src/index.ts
![image](https://user-images.githubusercontent.com/15182327/146315622-96c09581-d726-4381-82db-cf5abcafa315.png)
`IDToken` is exported but not used. When we try to use `IDToken` in some other files, should we just export `IDToken` from where it was defined?

Reply:
o it is being used by the react SDK. 

![image](https://user-images.githubusercontent.com/36393111/146335060-9d5d9a0f-a82f-4a30-804a-95f075ad79fe.png)

Will do